### ISSUE TITLE
Framework IMG_EWF_INFO has no member....

### ISSUE THREAD

**Issue by gj12** [sleuthkit/sleuthkit#768]
Hi 

I can't compile the framework

```
TskL01Extract.cpp: In function ‘void {anonymous}::ewf_image_close(TSK_IMG_INFO*)’:
TskL01Extract.cpp:77:39: error: ‘struct ewf::IMG_EWF_INFO’ has no member named ‘num_imgs’
             for (i = 0; i < ewf_info->num_imgs; i++) {
                                       ^~~~~~~~
TskL01Extract.cpp:78:32: error: ‘struct ewf::IMG_EWF_INFO’ has no member named ‘images’
                 free(ewf_info->images[i]);
                                ^~~~~~
TskL01Extract.cpp:80:28: error: ‘struct ewf::IMG_EWF_INFO’ has no member named ‘images’
             free(ewf_info->images);
                            ^~~~~~
TskL01Extract.cpp: In member function ‘TSK_IMG_INFO* TskL01Extract::openEwfSimple()’:
TskL01Extract.cpp:406:19: error: ‘struct ewf::IMG_EWF_INFO’ has no member named ‘num_imgs’
         ewf_info->num_imgs = a_num_img;
                   ^~~~~~~~
TskL01Extract.cpp:407:24: error: ‘struct ewf::IMG_EWF_INFO’ has no member named ‘images’
         if ((ewf_info->images = (TSK_TCHAR **) tsk_malloc(a_num_img * sizeof(TSK_TCHAR *))) == NULL)
                        ^~~~~~
TskL01Extract.cpp:412:24: error: ‘struct ewf::IMG_EWF_INFO’ has no member named ‘images’
         if ((ewf_info->images[0] =
                        ^~~~~~
TskL01Extract.cpp:417:28: error: ‘struct ewf::IMG_EWF_INFO’ has no member named ‘images’
         TSTRNCPY(ewf_info->images[0], ewfArchivePath, TSTRLEN(ewfArchivePath) + 1);
                            ^~~~~~
TskL01Extract.cpp:426:43: error: ‘struct ewf::IMG_EWF_INFO’ has no member named ‘images’
                 (char *const *) ewf_info->images,
                                           ^~~~~~
TskL01Extract.cpp:427:27: error: ‘struct ewf::IMG_EWF_INFO’ has no member named ‘num_imgs’
                 ewf_info->num_imgs, ewf::LIBEWF_ACCESS_FLAG_READ, &ewfError) != 1)
```

I tried different libewf versions (20140608, 20160424 and libewf master from github).

My system is Linux x64

**Comment by bcarrier** [comment]
This is a variation of the problem fixed in 28abf29085934c0e58bc28154f116763def76ca6. We changed the TSK_IMG_INFO structure and forgot to check the framework (we don't really use it anymore). 

Can you change the places with ewf_info->num_imgs errors to ewf_info->img_info.num_img and ewf_info->images to ewf_info->img_info.images and submit a pull request?  I'm not sure we have a system that we can easily build the framework on these days to verify it.

**Comment by uckelman-sf** [comment]
@simsong This can be closed, as the code in the bug report no longer exists.
